code/game/objects/items/weapons/id cards/station_ids.dm

```diff
@@ -147,7 +147,7 @@
 	job_access_type = /datum/job/captain
 
 /obj/item/weapon/card/id/gold/captain/spare
-	name = "colony director's spare ID"
+	name = "Colony Director's spare ID"
```

Comment:
If memory serves, this will make the game skip `the` in front of the name.
So it'll go from `This is the colony director's spare ID.` to `This is Colony Director's spare ID.`, which isn't really better, grammatically.

Reply:
Will add improper tag.